lib/Drush/CommandFiles/core/DrupliconCommands.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Drush\CommandFiles\core;
+
+class DrupliconCommands {
+
+  /**
+   * Print druplicon as post-command output.
+   *
+   * @hook post-command *
+   * @option $druplicon Shows the druplicon as glorious ASCII art.
+   */
+  public function druplicon($result, $argsAndOptions, $annotationData) {
+    if (isset($argsAndOptions['options']['druplicon'])) {
```

Comment:
Its a bit ugly, yes. Not that big of a dealbreaker for me though. I'm fine with whatever nastiness we have to do in the 8.x branch. In 9.x, I think we can break backwards compat and head toward a clean API.